sympy/functions/special/hyper.py

```diff
@@ -1100,6 +1100,19 @@ class appellf1(Function):
         \frac{(a)_{m+n} (b_1)_m (b_2)_n}{(c)_{m+n}}
         \frac{x^m y^n}{m! n!}.
 
+    Examples
+    ========
+
+    >>> from sympy.functions.special.hyper import appellf1
+    >>> appellf1(2., 1., 6., 4., 5., 6.)
+    0.0063339426292673
+    >>> appellf1(1, 2, 3, 5, 0.5, 0.25)
+    appellf1(1, 2, 3, 5, 0.5, 0.25)
```

Comment:
I don't think that the example to mix floats and rationals should be a model example.
We could probably have done to evaluate such cases numeric once there is there are any floats in the arguments.

Reply:
Sure. Will change

Reply:
At least one of the examples should show symbolic inputs.

Reply:
Also if it is possible to find more instructive inputs than just 1, 2, 3, 4, 5, 6 that would help as well.

Reply:
Done!